test/test_property.py

```diff
@@ -183,6 +183,10 @@ def test_value_append(self):
         self.assertEqual(len(p5), 2)
         self.assertRaises(ValueError, p5.append, "[a, b, c]")
 
+        p6 = Property('myprop', value=0, dtype=DType.int)
```

Comment:
Please move this snippet to `test_value` since strictly speaking its not an `append` test.